src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java

```diff
@@ -116,21 +116,21 @@ public boolean shouldComplete(NativeEvent event)
          Range range = getSession().getSelection().getRange();
```

Comment:
Thanks -- I think in this file my editor was mis-configured and automatically stripped whitespace :/